portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,57 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
+  //private ArrayList<String> comments; 
+  
+  /*
+  @Override
+  public void init(){
+    messages = new ArrayList<String>(); 
+    messages.add("Hello there");
+    messages.add("Roger Roger"); 
+    messages.add("Use the force");
+  }
+  */
+  
+  /*
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+    String json = convertToJsonUsingGson(comments);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+  */
+  
```

Comment:
Probably remove these for tidiness unless you're planning on putting them back in soon.